src/components/ButtonPanel.js

```diff
@@ -5,33 +5,33 @@ import Button from './Button';
 const ButtonPanel = () => (
   <div className="panel">
     <div className="first-row">
-      <Button btnName="Ac" />
-      <Button btnName="+/-" />
-      <Button btnName="%" />
-      <Button btnName="÷" />
+      <Button btnName="Ac" color="rgb(221, 221, 221)" false />
+      <Button btnName="+/-" color="rgb(221, 221, 221)" false />
+      <Button btnName="%" color="rgb(221, 221, 221)" false />
+      <Button btnName="÷" color="#FF8C00" false />
     </div>
     <div className="second-row">
-      <Button btnName="7" />
-      <Button btnName="8" />
-      <Button btnName="9" />
-      <Button btnName="x" />
+      <Button btnName="7" color="rgb(221, 221, 221)" false />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.